### PR TITLE
added boilerplate

### PR DESCRIPTION
**Pull request by vung1**
Added the boilerplate template.

### CODE REVIEW

**Comment by cqvu** [comment]
@vung1 Let's add a /docs directory to store our documentation and design files

**Comment by cqvu** [comment]
Also let's add the .env file to .gitignore to avoid accidentally pushing it

File: server/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (cqvu) [inline server/.gitignore:1]: Let's also add .env to the gitignore to avoid accidentally pushing it

Comment (vung1) [reply server/.gitignore:1]: Done

**Review by DualSaturn — APPROVED** [review]
Everything looks solid to me.

File: client/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```
Comment (cqvu) [inline client/.gitignore:12]: test

**Review by cqvu — CHANGES_REQUESTED** [review]
test

**Review by cqvu — APPROVED** [review]
Looks great!

**Pull request merged**